README.md

```diff
@@ -202,7 +202,32 @@ track.original # {}
 track.modified # { "title" => "Test", "body" => "Post", "comments" => [{ "_id" => "575fa9e667d827e5ed00000d", "title" => "test", "body" => "comment" }], ... }
 ```
 
-**Retrieving the list of tracked fields**
+**Whitelist the tracked attributes of embedded relations**
+
+If you don't want to track all the attributes of embedded relations in parent audit history, you can whitelist the attributes as below:
+
+```ruby
+class Book
+  include Mongoid::Document
+  ...
+  embeds_many :pages
+  track_history :on => { :pages => [:title, :content] }
```

Comment:
@dblock i was thinking of using the new syntax for hash. But to keep consistency with all the other documentation, i used the older one.